### PR TITLE
Add args parameter to custom queries

### PR DESCRIPTION
**Pull request by andrzejewsky**
closes: https://github.com/vuestorefront/vue-storefront/issues/5855

### CODE REVIEW

File: packages/core/docs/guide/composables.md [hunk 1]
```diff
@@ -434,4 +434,4 @@ module.exports = {
 };
 ```
 
-The custom query function always has in the arguments the default query and default variables and must return the query and its variables as well. In the body you can do anything you want with those parameters - you can override them or even change to the new ones.
+The custom query function always has in the arguments the default query (`query`), default variables (`variables`) and additional parameters (`args`) sent from the front-end. This function always must return the query and its variables as well, while in the body you can do anything you want with those parameters - you can override them or even change to the new ones.
```
Comment (filrak) [inline packages/core/docs/guide/composables.md:437]: can you add some example with a frontend call and then receiving this in query?

**Review by filrak — CHANGES_REQUESTED** [review]

File: packages/core/docs/guide/composables.md [hunk 2]
```diff
@@ -422,9 +427,10 @@ module.exports = {
       location: '@vue-storefront/commercetools-api/server',
       configuration: { /* ... */ },
       customQueries: {
-        'my-products-query': ({ query, variables }) => {
+        'my-products-query': ({ query, variables, metadata }) => {
```
Comment (filrak) [inline packages/core/docs/guide/composables.md:430]: make something more generic

Comment (filrak) [reply packages/core/docs/guide/composables.md:430]: `metaData` ?

**Review by filrak — APPROVED** [review]

**Comment by filipsobol** [comment]
Docs look fine :thumbsup:

**Pull request merged**